src/Swipeable.js

```diff
@@ -30,7 +31,8 @@ const Swipeable = React.createClass({
     return {
       flickThreshold: 0.6,
       delta: 10,
-      preventDefaultTouchmoveEvent: true
+      preventDefaultTouchmoveEvent: true,
+      component: 'div'
```

Comment:
I like `nodeName`, I used `component` due to conventions in other libraries, but I think some of those accept a react component. This won't accept a react component, there have been expressed needs for this to accept an actual react component, yet.